main.go

```diff
@@ -24,9 +24,9 @@ import (
 	"time"
 
 	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/google-osconfig-agent/config"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/google-osconfig-agent/guestpolicy"
```

Comment:
(it should point to this repo now)

Reply:
lol thanks sen.. testing failures is useful for us :)

Reply:
it's odd that prow even cannot clone this repo due to that error message :-\